pyseir/cli.py

```diff
@@ -38,6 +39,9 @@
 
 
 def _cache_global_datasets():
+    combined_datasets.build_us_latest_with_all_fields()
```

Comment:
Please add a comment explaining why we're calling them here and discarding the returned values.